lib/taurus/qt/qtcore/model/taurusdatabasemodel.py

```diff
@@ -534,6 +543,11 @@ class TaurusDbDeviceModel(TaurusDbBaseModel):
     def setupModelData(self, data):
         if data is None:
             return
+        try:
+            # TODO: Tango-centric
+            from taurus.core.tango.tangodatabase import TangoDatabase
+        except ImportError:
```

Comment:
Just for the curiosity, why this one has a different implementation: try...except?

Reply:
Good question...
I do not remember if handling the exception has a purpose or if it is just a leftover from different implementation attempts.
Nevertheless, I do not see it as harmful (other than the inconsistency). I propose to add a TODO to express this doubt and leave it as is